pyproject.toml

```diff
@@ -22,7 +22,7 @@ classifiers = [
 ]
 
 [tool.poetry.dependencies]
-python = "^3.7, <3.11"
+python = "^3.8, <3.11"
```

Comment:
Since we're (necessarily) dropping support for 3.7, let's update the "Getting Started with Scenic" page in the documentation correspondingly.

Reply:
I added a task on Todoist on this. Let me update that page when I finalize the installation with Makefile.